transcoder/src/keyframes.go

```diff
@@ -26,6 +27,9 @@ type KeyframeInfo struct {
 func (kf *Keyframe) Get(idx int32) float64 {
 	kf.info.mutex.RLock()
 	defer kf.info.mutex.RUnlock()
+	if len(kf.Keyframes) == 0 {
```

Comment:
Is this needed? The dummy keyframes should be created at this point, no?

Reply:
oops, forgot to remove this (for test purposes)